android/src/androidTest/java/org/ligi/gobandroidhd/uitest/TheGameInfoDialog.java

```diff
@@ -66,22 +67,22 @@ public void testThatFieldsWork() {
         final GameReviewActivity activity = getActivity();
         onView(withId(R.id.menu_game_info)).perform(click());
 
-        onView(withId(R.id.game_name_et)).perform(typeText(CUSTOM_GAME_NAME));
+        onView(withId(R.id.game_name_et)).perform(replaceText(CUSTOM_GAME_NAME));
```

Comment:
can you explain why typeText->replaceText ? The EditTexts should be empty - so IMHO typeText is correct